src/Entity/DecoupledAuthUser.php

```diff
@@ -54,6 +54,9 @@ public function isDecoupled() {
    * {@inheritdoc}
    */
   public function setDecoupled($decoupled = NULL) {
+    if (is_null($decoupled) && $this->isNew() && $this->name->value === NULL) {
+      $this->name->value = '';
+    }
     $this->decoupled = $decoupled ?: $this->name->value === NULL;
```

Comment:
I can see what you're trying to do here, but I'm not sure about it. Perhaps we're better explicitly setting the decoupled state as part of the entity build?  Alternatively well could get rid of the `postCreate` (and possibly `postLoad` also) and instead lazily figure it out in `getDecoupled` if not explicitly set? That might be better performance as well as we only figure it out when we need to.